### PR TITLE
update output of boot.LinuxImage

### PR DESCRIPTION
**Pull request by jensdrenhaus**

### CODE REVIEW

File: pkg/boot/linux.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (li *LinuxImage) String() string {
 }
 
 func copyToFile(r io.Reader) (*os.File, error) {
-	f, err := ioutil.TempFile("", "nerf-netboot")
+	f, err := ioutil.TempFile("", "linux-image")
```
Comment (rminnich) [inline pkg/boot/linux.go:60]: If we are kexec'ing a plan 9 kernel, doesn't the name nerf-netboot make more sense?

Comment (jensdrenhaus) [reply pkg/boot/linux.go:60]: I was only wondering if netboot is the right term. I integrated the OSImage interface in stboot some time ago (where there is a kind of local boot option as well using boot.LinuxImage) and it always felt strange to read "..netboot" in the Logs. Same must be true for grub parsers etc., I guess.
"linux-image" is just a suggestion.

Comment (rminnich) [reply pkg/boot/linux.go:60]: very good point. hmm. kexec-image?

Comment (hugelgupf) [reply pkg/boot/linux.go:60]: kexec-image would be good. I copied this code at some point from a netboot-only command, sorry for the name :)

**Comment by jensdrenhaus** [comment]
Reverted the insertion of newlines. Let ProgressReader take care about this. See #1960

**Review by hugelgupf — APPROVED** [review]

**Pull request merged**